apps/user-profile/src/components/modules/UserProfile/RecommendedIdeas/RecommendedIdeas.js

```diff
@@ -39,25 +39,46 @@ const RecommendedIdeas = ({ ideas }) => {
               flexWrap: 'wrap',
             }}
           >
-            {ideas.map((idea) => (
-              <RecommendedIdeaCardContainer key={Math.random()}>
-                <Card
-                  size="small"
-                  style={{ width: '90%', height: '17rem' }}
-                  noImage
-                  cardData={{
-                    id: idea.id,
-                    title: idea.ideaName,
-                    secondaryText: idea.tagline,
-                    tags: idea.skills?.map(({ skill }) => skill),
-                    description:
-                      idea.description.substring(0, 200) +
-                      (idea.description.length > 200 ? '...' : ''),
-                    href: `/ideaspace/workshopping/${idea.id}`,
+            
+            {ideas.slice(0, 4).map((idea) => {
+              const { level, description, title, skills, commitmentHoursPerWeek, project } = idea;
```

Comment:
Is there a reason your initialising these variables: "level, description, title, skills, commitmentHoursPerWeek, project" ?  I don't see them being used here.

Reply:
I will delete them and then create another pull request.